scripts/dev/update_3rdparty.py

```diff
@@ -110,14 +111,34 @@ def update_ace():
     urllib.request.urlcleanup()
 
 
-def run(ace=False, pdfjs=True, fancy_dmg=False, pdfjs_version=None):
+def test_dicts():
+    """Test available dictionaries."""
+    sys.path.insert(0, os.curdir)
+    from scripts import install_dict
+    from qutebrowser.config import configdata
+    configdata.init()
+    for lang in install_dict.available_languages():
+        sys.stdout.write('Testing dictionary {}... '.format(lang.code))
```

Comment:
I'd prefer `print(..., end='')` here - I'm not sure whether it makes any difference, but it's a common pattern for this kind of thing.